src/quick-lint-js/error.h

```diff
@@ -214,6 +214,13 @@
       .error(QLJS_TRANSLATABLE("'else' has no corresponding 'if'"),            \
              else_token))                                                      \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_else_with_conditional_missing_if, "E???",                          \
```

Comment:
Blocking: Pick a valid error code. The `???` should be three decimal digits. (I should write a test to ensure error codes are well-formed.)

Reply:
Done.